grr/server/grr_response_server/gui/ui/components/client_details/client_diff.ts

```diff
@@ -202,3 +202,59 @@ export function getClientVersions(clientSnapshots: Client[]):
 
   return clientChanges;
 }
+
+function getFirstStringsJoinedPath(path: any[]): string {
+  let tokens: string[] = [];
+  path.forEach(val => {
+    if (typeof val !== 'string') return;
+    tokens.push(val);
+  });
+
+  return tokens.join('.');
```

Comment:
Use functional programming for better readability:

```
return path.filter(val => typeof val === 'string').join('.')
```

Reply:
This method implementation was changed in the following PR: #842

Reply:
I updated the implementation here as well.